src/Resources/Pages/CreateRecord/Concerns/Translatable.php

```diff
@@ -57,7 +57,7 @@ protected function handleRecordCreation(array $data): Model
 
         foreach ($this->otherLocaleData as $locale => $localeData) {
             try {
-                $this->form->fill($this->form->getRawState());
+                $this->form->fill($this->form->getState());
                 $this->form->validate();
```

Comment:
Do you still need to validate since getState is already doing `$state = $this->validate();`?